docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[data-views-in-sec]]
+= {data-sources-cap} in Elastic Security
+
+The active {data-source} determines what data appears throughout {es-sec} on pages with event or alert data.
+{data-sources-cap} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+IMPORTANT: Custom indices may not be included in the <<default-data-view-security, default {data-source}>>. Modify it or create a custom {data-source} to include custom indices.
+
+[discrete]
+== Switch to another {data-source}
+
+You can tell which {data-source} is active by clicking the {data-source-cap} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, Timelines, or Hosts.
+To switch to another {data-source}, click **Choose {data-source}**, select one of the options, and click **Save**.
+
+image::images/dataview-select-gif.gif[gif of opening the data view selection menu]
+
+[discrete]
+== Create or modify a {data-source}:
```

Comment:
This section heading seems misleading -- it sounds like it's going to walk through a procedure, but the content is essentially just one sentence about permissions. Seems like this info should be in the introductory section, maybe as an admonition or prereqs.

```suggestion
== Create or modify a {data-source}
```

Reply:
I agree it doesn't walk through the steps, but it does link to them. I think it makes sense as its own section because users might find the page while trying to learn how to make a new data view, and the heading hopefully makes it easy to find while skimming. I don't think it would make sense as a prereq, because the page is an overview, not a tutorial, so there's no particular task described on the page to have prereqs. I do think it could work as admonition, but I'm not sure I see the downside to leaving it as-is, though I'm open to further feedback/suggestions.

Reply:
The section is a bit different after I incorporated some more feedback, so I'm curious what you think now.

Reply:
Revised section is better! I still have a few more suggestions that I'll add separately from this thread, but the section now does what the heading implies.